### PR TITLE
[8.0.0] Breaking change - changed the order of arguments in beforeColumnResize and beforeRowResize hooks

### PR DESCRIPTION
**Pull request by pnowak**
### Context
And for the sake of consistency, the same parameters changes the order in `afterColumnResize` and `afterRowResize` either.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #3328

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by wojciechczerniak** [comment]
@pnowak @swistach Any progress?

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js [hunk 1]
```diff
@@ -320,6 +320,44 @@ describe('manualColumnResize', () => {
...
+      manualColumnResize: true
+    });
+
+    expect(colWidth(spec().$container, 0)).toEqual(50);
+
+    hot.addHook('beforeColumnResize', () => 100);
+
+    hot.addHook('beforeColumnResize', () => 200);
+
+    hot.addHook('beforeColumnResize', () => {});
```
Comment (swistach) [inline src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]: Accordingly to typings in `d.ts` file, `beforeColumnResize` should return `number|void`.

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js [hunk 2]
```diff
@@ -320,6 +320,44 @@ describe('manualColumnResize', () => {
...
+
+    resizeColumn(0, 60);
+
+    expect(colWidth(spec().$container, 0)).toEqual(60);
```
Comment (swistach) [inline src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]: Shouldn't we get `200`? 🤔

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js [hunk 3]
```diff
@@ -320,6 +320,44 @@ describe('manualColumnResize', () => {
...
+      colHeaders: true,
+      manualColumnResize: true
+    });
+
+    expect(colWidth(spec().$container, 0)).toEqual(50);
+
+    hot.addHook('beforeColumnResize', () => 100);
+
+    hot.addHook('beforeColumnResize', () => 200);
+
```
Comment (swistach) [inline src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]:
```suggestion
```

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js [hunk 4]
```diff
@@ -320,6 +320,44 @@ describe('manualColumnResize', () => {
...
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetData(3, 3),
+      colHeaders: true,
+      manualColumnResize: true
+    });
+
+    expect(colWidth(spec().$container, 0)).toEqual(50);
+
+    hot.addHook('beforeColumnResize', () => 100);
+
```
Comment (swistach) [inline src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]:
```suggestion
```

File: src/plugins/manualRowResize/test/manualRowResize.e2e.js [hunk 5]
```diff
@@ -133,6 +133,43 @@ describe('manualRowResize', () => {
...
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetData(3, 3),
+      rowHeaders: true,
+      manualRowResize: true
+    });
+
+    expect(rowHeight(spec().$container, 0)).toEqual(24);
+
+    hot.addHook('beforeRowResize', () => 100);
+
```
Comment (swistach) [inline src/plugins/manualRowResize/test/manualRowResize.e2e.js]:
```suggestion
```

File: src/plugins/manualRowResize/test/manualRowResize.e2e.js [hunk 6]
```diff
@@ -133,6 +133,43 @@ describe('manualRowResize', () => {
...
+    hot.addHook('beforeRowResize', () => 200);
+
```
Comment (swistach) [inline src/plugins/manualRowResize/test/manualRowResize.e2e.js]:
```suggestion
```

File: src/plugins/manualRowResize/test/manualRowResize.e2e.js [hunk 7]
```diff
@@ -133,6 +133,43 @@ describe('manualRowResize', () => {
...
+    hot.addHook('beforeRowResize', () => {});
```
Comment (swistach) [inline src/plugins/manualRowResize/test/manualRowResize.e2e.js]: Accordingly to typings in d.ts file, `beforeRowResize ` should return `number|void`.

File: src/plugins/manualRowResize/test/manualRowResize.e2e.js [hunk 8]
```diff
@@ -133,6 +133,43 @@ describe('manualRowResize', () => {
...
+
+    resizeRow(0, 60);
+
+    expect(rowHeight(spec().$container, 0)).toEqual(61);
```
Comment (swistach) [inline src/plugins/manualRowResize/test/manualRowResize.e2e.js]: Why we got `61` if we want to set `60`? And why that row is not `200`px height? 🤯

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js (hunk 2 above)
Comment (pnowak) [reply src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]: Our helper `resizeColumn` simulate `mouseup` functionality. So it uses in some sense`onMouseUp` method. This method doesn't use the value returned from `beforeColumnResize`. 
If we make use `double click` ( `afterMouseDownTimeout` method which uses the returned value from this hook) we get `200`. 
So should I change this test scenario to make use of `dbclick`?

File: src/plugins/manualRowResize/test/manualRowResize.e2e.js (hunk 8 above)
Comment (pnowak) [reply src/plugins/manualRowResize/test/manualRowResize.e2e.js]: Same as above.
I think this extra `1px` is from border from `tr:first-child td`. And this surplus is common, e.g. - https://github.com/handsontable/handsontable/blob/master/src/plugins/manualRowResize/test/manualRowResize.e2e.js#L59

File: src/plugins/manualColumnResize/test/manualColumnResize.e2e.js (hunk 2 above)
Comment (swistach) [reply src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]: Ok, I see that only fake `dblclick` uses `afterMouseDownTimeout` where we can overwrite the final value. This inconsistency should be done in a separated issue.
But here, if we would like to test column/row size after dblclick then we need a test for that.

Comment (swistach) [reply src/plugins/manualColumnResize/test/manualColumnResize.e2e.js]: Everything is clear. I didn't notice last commit. Now it looks ok :+1:

**Pull request merged**